src/main/java/com/mraof/minestuck/item/MSItems.java

```diff
@@ -502,403 +502,403 @@ public static void registerItems(final RegistryEvent.Register<Item> event)
 	{
 		IForgeRegistry<Item> registry = event.getRegistry();
 		
-		registerItemBlock(registry, BLACK_CHESS_DIRT, MSItemGroup.MAIN);
-		registerItemBlock(registry, WHITE_CHESS_DIRT, MSItemGroup.MAIN);
-		registerItemBlock(registry, DARK_GRAY_CHESS_DIRT, MSItemGroup.MAIN);
-		registerItemBlock(registry, LIGHT_GRAY_CHESS_DIRT, MSItemGroup.MAIN);
-		registerItemBlock(registry, SKAIA_PORTAL, new Item.Properties().tab(MSItemGroup.MAIN).rarity(Rarity.EPIC));
+		registerItemBlock(registry, BLACK_CHESS_DIRT.get(), MSItemGroup.MAIN);
+		registerItemBlock(registry, WHITE_CHESS_DIRT.get(), MSItemGroup.MAIN);
+		registerItemBlock(registry, DARK_GRAY_CHESS_DIRT.get(), MSItemGroup.MAIN);
+		registerItemBlock(registry, LIGHT_GRAY_CHESS_DIRT.get(), MSItemGroup.MAIN);
+		registerItemBlock(registry, SKAIA_PORTAL.get(), new Item.Properties().tab(MSItemGroup.MAIN).rarity(Rarity.EPIC));
```

Comment:
I'd suggest that you instead update `registerItemBlock()` to take a `Supplier<Block>`

Reply:
Not sure. I suppose it doesn't matter that much as long as that is on the way.